### ISSUE TITLE
multiple 'js-' classes on a single el

### ISSUE THREAD

**Issue by baptistebriel** [dcamilleri/query-dom-components#3]
Hello, it's is me again :)

So, I have this use case; I'd like to apply a css prop on all `.js-el` elements and after that, apply a tween on all `.js-stagger` elements.

``` html
<div class="js-el js-stagger"></div>
<div class="js-el js-stagger"></div>
<div class="js-stagger"></div>
```

**Currently, you cannot have multiple `js-` classes on a single element.**

The expected object would therefore be: 

``` javascript
var object = {
   el: NodeList[2],
   stagger: NodeList[3]
};
```

Let me know what you think!

**Comment by dcamilleri** [comment]
Hey Baptiste, 

In fact, the way `query-dom-components` is build is to target a whole `component` and fetch all its `sub-components`. To match your example, you can just do the following: 

```html
<div class="js-comp"></div>
    <div class="js-comp-el js-comp-stagger"></div>
    <div class="js-comp-el js-comp-stagger"></div>
    <div class="js-comp-stagger"></div>
</div>
```

And then apply queryDom to it: 

```javascript
var dom = queryDom({
    el: document.querySelector('.js-comp'),
    prefix: 'js-comp-'
});
```

Then you should get 
```javascript
dom = {
    el: NodeList[2],
    stagger: NodeList[3]
};
```

Is that what you try to do ?

**Comment by dcamilleri** [comment]
Hey Baptiste ! 

You're right, there's an issue with multiple classes on a single element.
I'll fix it ASAP !

**Comment by baptistebriel** [comment]
Don't worry, no rush!

Cheers mate :)

**Comment by dcamilleri** [comment]
Hey @baptistebriel !
Should be fixed by now, it's up on v0.1.0.
Let me know if you still have issues with it.
